### ISSUE TITLE
PAW F.E.: Build (code) ONE results screen R1 on an a11y-friendly platform (Staging)

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#14729]
### User story

**AS A** Veteran
**I WANT** to usability-test the PACT Act Wizard results screens on live, accessible code
**SO THAT** I can ensure the best possible version of those screens (with accordions and heading levels) will be usable.

### Engineering notes / background

Resources for content:
- Content doc with all questions and results screens: on [Sharepoint (CAG)](https://dvagov-my.sharepoint.com/:w:/g/personal/cynthia_merrill_va_gov/EXcXBeLPnthDhzKkq0tHOgkB1hz4H5z-TpljngwAkGniFw?e=WJhfZI)
  - See Cindy for questions about the content
  - Some details in the content may change

Possible platforms:
- CodePen
  - Risks around learning curve, a11y, possibility that functionality just isn't possible, churn around prototype UX
  - screen reader experience may include some CP content/landmarks (question for LauraF)
  - dev effort: 1-2 sprints
- VA.gov Staging
  - Risks: procedural (e.g. IA approval), but we could start building ahead of actual URL approval
  - dev effort: <1 sprint

Team recommendation:
- Immediately pursue IA approval for this plan, which is the same thing we did with the New Homepage research

Assumption:
- We'll just build this one first. We will probably build another version subsequently.
- We estimated 8 assuming Staging approach

## Acceptance criteria

>Consider
> - Design / Accessibility reviews
> - Collab cycle requirements
> - Device sizes (mobile first)
> - Documentation updates / Change management
< - [Content model documentation](https://prod.cms.va.gov/admin/structure/cm_document)
> - Testing notes

- [x] Both screens of the R1 "results screen flow" are built, and linked by a link at the bottom of the first screen
- [x] The first screen is available at [STAGING URL TBD]
  - [x] Note: IA team will give us a base URL, we should come up with our own convention for URLs for pages/screens under it (https://github.com/department-of-veterans-affairs/va.gov-team/issues/63472)
- [x] This URL is hidden from production visibility using the app registry in content-build [like we did for Income Limits](https://github.com/department-of-veterans-affairs/content-build/blob/7c7a8984b85ce900e0f98393be07216e4562b0b5/src/applications/registry.json#L1439)
- [x] Confirmed that the screens conform to the design system (and are "working right") on both mobile and desktop devices
- [x] Research review required (CindyM)
- [x] a11y review required (LauraF)
- [ ] ~~CAIA review required (Danielle Thierry)~~ now tracked in #15013

**Comment by wesrowe** [comment]
To address the IA-team risk, I wrote a slack message in the Content-IA channel that will post Thursday morning, 8/10. Hoping to hear they're fine with taking the same approach as New Homepage prototyping.

**Comment by cindymerrill** [comment]
@wesrowe @randimays Will this coded results screen be responsive? If we need to pick a format, Danielle would like a mobile view instead of desktop.

**Comment by randimays** [comment]
Adding content per Cindy's doc is complete.

Three PRs are in draft waiting for final URLs and design guidance. These establish the boilerplate for the new application and add the Results set 1, pages 1-2.

- [devops PR for allowing client-side routing](https://github.com/department-of-veterans-affairs/devops/pull/13330)
- [content-build PR for registering the new application](https://github.com/department-of-veterans-affairs/content-build/pull/1665/)
- [vets-website PR for adding application code](https://github.com/department-of-veterans-affairs/vets-website/pull/25237)

**Comment by jilladams** [comment]
From planning: this is a goal for full team press; acknowledge that we've got a dependency on CAIA so there's some risk, DaveC has signed off on that.

**Comment by randimays** [comment]
Official url for the staging prototype: `/test/pact-act/results/#`

Note: the `noindex` tag will be automatically added by the content-build configuration that keeps the app from going to prod.

CAIA ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/63472

**Comment by randimays** [comment]
Official URL is plugged in. Content and design are complete. Waiting for a `devops` deploy to enable client-side routing so I can prevent deep-linking / fix 404s. Will post for the team to review on Monday.

**Comment by randimays** [comment]
Tried to get this merged in today, but I'm blocked by [a devops change](https://github.com/department-of-veterans-affairs/devops/pull/13330/files) I made not propagating across environments. I need this reflected locally so I can enable client-side routing. This way, you won't get a 404 for reloading on either of the results pages. Hoping to get this resolved tomorrow so I can merge in.

**Comment by randimays** [comment]
Update for mid-sprint: Posted [the PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25237) for review. Not anticipating carryover here unless we can't get the Staging build reviewed by everyone in time.

**Comment by randimays** [comment]
Update: added [a new PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25380) for opening informational links in a new tab (on Results page 2).

**Comment by randimays** [comment]
We have approval from Jordan, Laura and Cindy. Waiting to see if Danielle has any feedback. I suppose this ticket could close by EOD either way, and we can always adjust per Danielle's feedback when we receive it?

**Comment by cindymerrill** [comment]
Makes sense. We might not receive any feedback from Danielle until she reviews the entire prototype that I'm building, which links to the R1 results page. This isn't going to happen until next sprint.

**Comment by jilladams** [comment]
I'm ok saying: let's ticket / track Danielle's feedback and integrating it separately. cc @wesrowe

**Comment by randimays** [comment]
@jilladams Do you want me to create the new ticket for feedback?

**Comment by cindymerrill** [comment]
Danielle may have feedback on the R1 screen, the visual interaction flow, and/or the entire PAW prototype when it's more built

**Comment by cindymerrill** [comment]
I pinged Danielle again just now [in slack](https://dsva.slack.com/archives/C52CL1PKQ/p1693343490944479?thread_ts=1693250181.301529&cid=C52CL1PKQ), and this time she reacted with emojis. Can't tell if she plans to provide some feedback soon, or wait until I share the entire prototype with her. @randimays @jilladams

**Comment by jilladams** [comment]
I made a very stubby ticket to implement Content team prototype feedback: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15013
